### DISCUSSION TITLE
Unknown var specified in a var shortcode.

### DISCUSSION THREAD

**Discussion by rputikar** [quarto-dev/quarto-cli#4850] [Q&A]
I am trying to render README.qmd file that is built using vars. 

_variables.yml:

```
author: John Doe
```

README.qmd 

```
---
format: gfm
---
author: {{< var author >}}
``` 

when I try to render README.qmd, I get following warning
WARNING: Unknown var author specified in a var shortcode.

```
quarto render .\README.qmd
pandoc
  to: >-
    commonmark+autolink_bare_uris+emoji+footnotes+gfm_auto_identifiers+pipe_tables+strikeout+task_lists+tex_math_dollars
  output-file: README.md
  standalone: true
  default-image-extension: png

WARNING: Unknown var author specified in a var shortcode.
Output created: README.md
```

Generated README.md

```
author: **?var:author**
```
If I rename the file to readme.qmd it generates  a readme.md in _site folder with proper content. 
```
quarto render .\readme.qmd
pandoc
  to: >-
    commonmark+autolink_bare_uris+emoji+footnotes+gfm_auto_identifiers+pipe_tables+strikeout+task_lists+tex_math_dollars
  output-file: readme.md
  standalone: true
  default-image-extension: png

Output created: _site\readme.md
```

```
author: John Doe
```
How do I access the variables in README.qmd?

**Comment by lucahammer** [comment]
I don't know why, but after I added `version: 1.2` in the first line of the _variables.yml file from the example in the documentation, it started to work for me. Even after removing it again.

**Comment by mcanouil** [comment]
To use `_variables.yml`, your document must be inside a Quarto project.
A quarto project is defined by the presence of `_quarto.yml`.
